ledger-obsolete/src/main/kotlin/net/corda/v5/ledger/obsolete/LedgerGroupParameters.kt

```diff
@@ -1,10 +1,19 @@
 @file:JvmName("LedgerGroupParameters")
+
 package net.corda.v5.ledger.obsolete
 
 import net.corda.v5.membership.GroupParameters
-import net.corda.v5.membership.getValue
 
-const val NOTARIES_KEY = "corda.notaries"
+private const val NOTARIES_KEY = "corda.notary"
```

Comment:
Why is this not plural? We need to allow multiple notaries in the network

Reply:
Reverted. Originally changed to match design doc but plural would be more in line with other properties.